userspace/libsinsp/container_info.h

```diff
@@ -36,6 +36,7 @@ enum sinsp_container_type
 	CT_CUSTOM = 5,
 	CT_CRI = 6,
 	CT_CONTAINERD = 7,
+	CT_BPM = 8,
```

Comment:
We're working on CRI-o container support and have already reserved 8 internally for it. Can you please change the enum value to 9 here?